src/backends/fixerio/index.js

```diff
@@ -60,26 +51,15 @@ class FixerIoBackend {
       })
       apiData = result.body
     }
-
+    const pairs = config.get('tradingPairs')
```

Comment:
Is this guaranteed to be set? Even if so it might be good to check here to make sure.